### PR TITLE
Add $BASED

### PR DESCRIPTION
**Pull request by cryptoandy11**

### CODE REVIEW

File: spaces/based/skin.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.based {
+  --primary-color: #F7ACB2;
+  --header-bg: #296788;
+  --bg-color: #296788;
+  --border-color: #fff;
+  --text-color: #fff;
+  --link-color: #fff;
```
Comment (e-asia) [inline spaces/based/skin.scss:7]: @cryptoandy11 The published button is written in white on a white background, the style should be changed.

Comment (cryptoandy11) [reply spaces/based/skin.scss:7]: Not sure what you are talking about.  Are you saying `link-color` and `text-color` should be different than `border-color`?  swerve style has light colors for text and link
Screen Shot 2020-09-12 at 1 54 13 AM
 except a dark border color

Comment (e-asia) [reply spaces/based/skin.scss:7]: image
Maybe you could use a dark blue on `link-color`?

Comment (cryptoandy11) [reply spaces/based/skin.scss:7]: just changed it.

Comment (cryptoandy11) [reply spaces/based/skin.scss:7]: anything else I need to do @e-asia ?

**Review by bonustrack — APPROVED** [review]

**Pull request merged**